### DISCUSSION TITLE
Noayo: Orchard Core hosting platform

### DISCUSSION THREAD

**Discussion by agriffard** [OrchardCMS/OrchardCore#8723] [Show and tell]
If you look for a way to have an Orchard Core instance running online with a nice Bootstrap theme, please visit https://noayo.com

Noayo is an Orchard Core site that allows you to subscribe and get your own Orchard Core tenant in exchange of a monthly fee (5$ and 10$ plans are available).

Note: 20% of the earnings will be redistributed among the main Orchard Core contributors.

**Comment by hishamco** [comment]
Is Noayo dead? I have never heard anything about it since 2021
